src/snowflake/snowpark/modin/plugin/_internal/transpose_utils.py

```diff
@@ -47,7 +47,12 @@ def transpose_empty_df(
     return SnowflakeQueryCompiler.from_pandas(
         native_pd.DataFrame(
             columns=original_frame.index_columns_index,
-            index=original_frame.data_columns_index,
+            index=original_frame.data_columns_index
```

Comment:
try_convert_index_to_native?

Reply:
This causes circular import error if I try to import `try_convert_index_to_native` in this class

Reply:
It worked for me:

```
from snowflake.snowpark.modin.pandas.utils import try_convert_index_to_native

return SnowflakeQueryCompiler.from_pandas(
    native_pd.DataFrame(
        columns=original_frame.index_columns_index,
        index=try_convert_index_to_native(original_frame.data_columns_index),
    )
)
```

Reply:
Hmm weird, I think if i import within the function it works but I was trying to import at the top of the file yesterday. Just updated it now